docs/tildagon-apps/guides/ui-widgets.md

```diff
@@ -0,0 +1,232 @@
+# UI Widget Overview
+
+You can use the following premade [`app_components`](https://github.com/emfcamp/badge-2024-software/blob/main/modules/app_components/menu.py) to create user interfaces:
+
+## Menu
+
+The [`Menu`](https://github.com/emfcamp/badge-2024-software/blob/main/modules/app_components/menu.py) component, allows you to create menus with which you can scroll through a list of options and select an item.
+
+### Example
+
+You can see how these components work in the [`menu_demo.py`](https://github.com/emfcamp/badge-2024-software/blob/main/modules/firmware_apps/menu_demo.py).
+
+### Usage
+
+## Notification
+
+The [`Notification`](https://github.com/emfcamp/badge-2024-software/blob/main/modules/app_components/notification.py) component allows you to create pop up notifications.
+
+### Example
+
+This example app shows a notification for each button you press.
+
+```python
+from app import App
+from app_components import Notification, clear_background
+from events.input import Buttons, BUTTON_TYPES
+
+
+class NotificationDemo(App):
```

Comment:
What do you think about consolidating all these demos into a single UI-Widget Demo app?